dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -6,13 +6,14 @@
 # the final image because of permission changes.
 FROM {{ .buildFrom }} AS home
 
+{{- if contains .buildFrom "wolfi" }}
 RUN for iter in {1..10}; do \
-        apt-get update -y && \
-        DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends --yes libcap2-bin && \
```

Comment:
Is this no longer needed on the Ubuntu images?

Reply:
It's an image for building the file structure, not the final image. I defined the builder image as Wolfi instead of Ubuntu, since it's more compact and reduces the build time.